src/basic/efivars.c

```diff
@@ -96,6 +96,23 @@ int efi_get_variable(
                                 (void) usleep_safe(EFI_RETRY_DELAY);
                 }
 
+                /* According to comment of linux source code, efivarfs represents uncommitted
```

Comment:
we should not reference the concept of "uncommited" variables in a comment here, if noone knows what that precisely is supposed to be.

Maybe somebody can ping the person who added the comment about this to the kernel and ask for clarification what that is supposed to be?

Reply:
I'll wait for the clarification to update the comment.